### ISSUE TITLE
anhydramnios

### ISSUE THREAD

**Issue by sparkste** [obophenotype/human-phenotype-ontology#7003]
**Preferred term label:**
anhydramnios

**Synonyms**

**Definition (free text, please give PubMed ID)**
no amniotic fluid around a fetus

**Parent term (use hpo.jax.org/app)**
anhydramnios

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Potter syndrome
Multinucleated neurons, anhydramnios, renal dysplasia, cerebellar hypoplasia, and hydranencephaly
Meckel syndrome
Renal hypoplasia/aplasia, renal agenesis
Megacystis-microcolon-intestinal-hypoperistalsis syndrome
Denys-Drash syndrome
Others

**Your nano-attribution (ORCID)**
0000-0002-85932186

**Comment by patt1morgan** [comment]
@RoniZemet, @saintfrison.  Thank you for agreeing to work with us to advance the perinatal section of the HPO. We would appreciate it if you could review this new term request and approve it or help us improve it. Any suggestions welcome!

**Comment by saintfrison** [comment]
Here are a few of my ideas.

Definition : no amniotic fluid around a fetus  (US :deepest vertical pocket of amniotic fluid ≤ 2 cm in the second trimester3)

Potter sequence : ie
	Potter facies (distinctive facial features including a microretrognathia ; a flattened, depressed bridge of the nose; low-set flat ears ;abnormally prominent skins folds).
	The lungs may be hypoplastic (when anydramnios occurs before 24GW)
	Positional extremity deformities (club feet, camptodactyly,...)


Consequence of :
	Congenital anomalies of the kidney and urinary tract (CAKUT) (renal agenesis, severe cystic kidney disease, and lower urinary tract obstruction)
	Maternal drug use in pregnancy (NSAIDs -  angiotensin II receptor type 1 (AT1) antagonist, responsible for renal tubular dysgenesis) 
	Placenta : PROM

I’m very sorry not to be able (again) to attend the 23/11 worshop.
I hope I can join you some other time.

**Comment by pnrobinson** [comment]
Thanks, added term
